packages/design-tokens/package.json

```diff
@@ -17,14 +17,30 @@
   ],
   "private": false,
   "license": "MIT",
+  "sideEffects": false,
   "scripts": {
+    "test": "yarn jest",
     "build": "yarn clean && yarn prepublish",
+    "build:json": "yarn ts-node --files ./bin/buildCssVarTokens.ts",
     "build:less": "json-to-flat-sass './tokens/*.json' 'less' --extension 'less' --caseType 'kebab' && prettier less/* --write",
     "build:sass": "json-to-flat-sass './tokens/*.json' 'sass' --extension 'scss' --caseType 'kebab' && prettier sass/* --write",
-    "clean": "rimraf 'dist/**' 'tokens/**/*.js' 'tokens/**/*.d.ts' 'tokens/**/*.map'",
-    "prepublish": "yarn build:less && yarn build:sass"
+    "clean": "rimraf 'dist/**' './(!node_modules)/**/*.js' 'tokens/**/*.d.ts' 'tokens/**/*.map' ",
+    "prepublish": "yarn build:json && yarn build:less && yarn build:sass"
   },
   "devDependencies": {
-    "json-to-flat-sass": "^1.0.0"
+    "json-to-flat-sass": "^1.0.0",
+    "yargs": "16.2.0",
+    "@types/yargs": "16.0.0",
+    "prettier": "2.2.1",
```

Comment:
Do you need all of these dependencies? I would expect prettier/ts-node to use the workspace root packages.

Reply:
@ActuallyACat   i added them because eslint was complaining about extraneous dependencies...should i ignore that?

Reply:
Oh, I see - my mistake. For packages that are _only_ used in `package.scripts` I just let those default to whatever is version is defined [here](https://github.com/cultureamp/kaizen-design-system/blob/master/package.json). You're not just using prettier in `package.scripts`, you're also using it [here](https://github.com/cultureamp/kaizen-design-system/pull/1030/files#diff-053c9716520eec2ec0f883918db9c2acab7d818654cc4209189f0c9b457253d1R3). So what you have here is valid.